uber/templates/preregistration/stripeForm.html

```diff
@@ -5,7 +5,8 @@
         data-key="{{ c.STRIPE_PUBLIC_KEY }}"
         {% if email %}data-email="{{ email }}"{% endif %}
         data-zip-code="true"
```

Comment:
The ``{{ charge.amount }}`` whitespace is just a style change.

``data-allow-remember-me`` is a new Stripe feature we've been wanting for years, since our public laptops have a tendency to remember people's credit card info, which seems bad.  We've disabled that by going into Firefox and tweaking the cookie settings, but sometimes those get reset, so this is a much more robust solution.